xmeffects.js

```diff
@@ -56,7 +56,7 @@ function eff_t1_3(ch) {  // portamento
 
 function eff_t0_4(ch, data) {  // vibrato
   if (data & 0x0f) {
-    ch.vibratodepth = data & 0x0f;
+    ch.vibratodepth = (data & 0x0f) * 2;
```

Comment:
Listening to the test case xm-file [here](http://resources.openmpt.org/player_tests/xm/VibratoWaveforms.xm) using different values made me choose 2. I have no actual technical proof for why that is :)